src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -67,12 +69,16 @@ public void testExternalUserScenario() {
         findUserStatusByEmailScenarios();
         modifyRolesScenarios();
         suspendUserScenarios();
+        findMFAScenario();
```

Comment:
dont call this fro here , otherwise this test test will be called twice, since @Test will invoke this method again